### ISSUE TITLE
as PII is retained, an end user needs to be able to 'subscribe' to changes in the Provacy Policy

### ISSUE THREAD

**Issue by herrold** [5calls/5calls#343]
This is currently in 'static' content, but needs to have something like:

> email me when this changes at my address: [ whoever@example.com ]

facility.  I only notices when a CO and diff pointed me at a change on the page:

>  ./5calls/static/html/privacy.html

**Comment by nickoneill** [comment]
Thanks @herrold - I think it's worth noting that although our Privacy Policy states that we may retain PII, we don't store/save any of it right now.

Are you interested in implement a subscribe for privacy changes? I don't anticipate a significant number of them but it's good to have for people interested in this sort of thing.

**Comment by herrold** [comment]
I see two sets of PII disclosed or sought here:  
1. Long/Lat, and 
1. address information

see image at: 
   http://gallery.herrold.com/stuff/5calls-PII-in-flight.png

**Comment by herrold** [comment]
@nickoneill  asked: 
> Are you interested in implement a subscribe for privacy changes? I don't anticipate a significant number of them but it's good to have for people interested in this sort of thing.

Sure -- I'll look

**Comment by nickoneill** [comment]
Right. Used to determine your reps via the Google Civic API and then discarded. We request it, that's why it's in the Privacy Policy, but FWIW don't store it.

**Comment by herrold** [comment]
so PII in flight, and PII handed to a third party not covered by an agreement to honor your Privacy Policy, it would seem

I cannot see how to 'assign' this issue to myself at GH -- can you assist me please?

**Comment by nickoneill** [comment]
I just assigned.

We already have a Mailchimp list for weekly emails. It would probably make sense to be able to submit emails via the same mechanism to a new list for privacy updates. See the footer for how that's accomplished currently.

I created a new list for just this. Here's the new URL that the form should POST to: //5calls.us16.list-manage.com/subscribe/post?u=82a164d5fe7f51f4a4efb1f83&amp;id=b4bc52bf1e

**Comment by herrold** [comment]
FWIW, that Lat / Long 
> "loc": "37.7510,-97.8220",
 seems to be a 'privacy protecting' null value -- I see it here as well
   https://www.codeproject.com/articles/1105492/using-google-maps-api
(I recognized the map extract)

As such if it is hit, we know that particular Lat / Long address is not likely to be useful
